src/test/java/com/spotify/logging/LoggingConfiguratorTest.java

```diff
@@ -48,4 +56,50 @@ public void testGetSyslogAppender() {
 
   }
 
+  private String getLoggingContextHostnameProperty() {
+    final Logger accessPointLogger = (Logger) LoggerFactory.getLogger("logger");
+    final LoggerContext loggerContext = accessPointLogger.getLoggerContext();
+    return loggerContext.getProperty("hostname");
+  }
+
+  @Test
+  public void shouldReturnHeliosHostnameWithNoHostname() {
+    LoggingConfigurator.configureDefaults();
+    assertNull(getLoggingContextHostnameProperty());
```

Comment:
This check should perhaps be changed to something like 'assert that the loggerContext doesn't have the hostname key'.